frontend/src/scenes/billing/v2/billingLogic.ts

```diff
@@ -96,9 +96,10 @@ export const billingLogic = kea<billingLogicType>([
         billingAlert: [
             (s) => [s.billing],
             (billing): BillingAlertConfig | undefined => {
-                console.log(billing?.products)
-
-                const productOverLimit = billing?.products.find((x) => {
+                if (!billing) {
+                    return
+                }
+                const productOverLimit = billing.products.find((x) => {
```

Comment:
Why is this in this PR?

Reply:
Ah this is just a quick fix for a console log. Didn't realise it was this PR but safe to go in